src/models/elan1-r1/README.md

```diff
@@ -1,29 +1,41 @@
 # Eland 1U (elan1-r1)
 
-![Eland 1U](./img/elan1-r1-frontports.avif)
+![Eland 1U](./img/elan1-r1-frontports.webp)
 
 The System76 Eland 1U is a rack-mount server with the following specifications:
 
-- Operating System
-    - Ubuntu Server 22.04 LTS (64-bit)
 - CPU options
-    - Supports AMD EPYC™ 7003 series processor family
-    - Single processor
+  - Supports AMD EPYC™ 7003 series processor family
+  - Single processor
 - Memory
-    - Up to 4096GB (16x256GB) ECC DDR4 @ 3200 MHz
+  - Up to 4096GB (16x256GB) ECC DDR4 @ 3200 MHz
 - Networking
-    - 2x 1GbE LAN ports (1 x Intel® I350-AM2)
-    - 1x 10/100/1000 management LAN
+  - 2x 1GbE LAN ports ([1 x Intel® I350-AM2](https://ark.intel.com/content/www/us/en/ark/products/52968/intel-ethernet-controller-i350am2.html))
+  - 1x 10/100/1000 management LAN
 - Expansion
-    - 1 x PCIe x16 slot (Gen4)
+  - 1x PCIe 4.0 x16
 - Ports
-    - Front: 2 x USB 3.0, 1 x Power button with LED, 1 x ID button with LED, 1 x NMI button, 1 x Reset button, 2 x LAN activity LEDs, 1 x HDD activity LED, 1 x System status LED
-    - Rear: 3 x USB 3.0, 1 x VGA, 1 x COM, 2 x RJ45, 1 x MLAN, 1 x ID button with LED
+  - Front: 2 x USB 3.0
+    - 1 x Power button with LED
+    - 1 x ID button with LED
+    - 1 x NMI button
+    - 1 x Reset button
+  - Rear: 3 x USB 3.0
+    - 1 x VGA, 1 x COM
+    - 2 x RJ45, 1 x MLAN
+- LED headers
+  - Front:
+    - 2 x LAN activity LEDs
+    - 1 x HDD activity LED
+    - 1 x System status LED
+  - Rear:
+    - 1 x ID button with LED
 - Power Supply
-    - 2 x 650W redundant PSUs, 80 PLUS Platinum
+  - 2 x 650W redundant PSUs, 80 PLUS Platinum
+  - AC Input: 100-240Vac/ 10-5A, 50-60Hz
 - Dimensions
-    - 17.2” × 26” × 1.7” (43.8cm × 66.0cm × 43.5cm)
+  - 43.8cm × 66.0cm × 43.5cm
 - Weight
-    - 44 lbs (17.24 kg)
+  - 17.24 kg
 - Model
-    - elan1-r1 - [Technical Documentation](./elan1-r1_manual.pdf)
+  - elan1-r1 - [Gigabyte R152-Z3{1,2,3}](./chassis-manual.pdf)
```

Comment:
I'd like to keep the pdf's filename with the S76 model name so users can keep it organized better if they download it.